jboss-image-streams.json

```diff
@@ -1193,10 +1210,9 @@
                         "annotations": {
                             "description": "Red Hat JBoss Data Virtualization 6.3 S2I images.",
                             "iconClass": "icon-jboss",
-                            "tags": "datavirt,jboss,xpaas",
-                            "supports":"datavirt:6.3,xpaas:1.4",
-                            "version": "1.2",
-                            "openshift.io/display-name": "Red Hat JBoss Data Virtualization 6.3"
```

Comment:
Did we intend to remove the display-name?

Reply:
no, me sloppy